src/java/org/apache/cassandra/locator/InetAddressAndPort.java

```diff
@@ -382,4 +385,83 @@ public long serializedSize(InetAddressAndPort from, int version)
             }
         }
     }
+
+    // Serializer for handling FWD_FRM message parameters. Pre-4.0 deserialization is a special
+    // case in the message
```

Comment:
nit: Both this and the comments for `Serializer` above could be in proper JavaDoc format. (Would allow for links and such, etc.)